public/assets/css/style.css

```diff
@@ -115,9 +115,9 @@ button {
 .svg {
     position: fixed;
     z-index: 6;
-    transform: scaleY(0.8);
-    margin-left: 12rem;
     margin-top: 2rem;
+    transform: scale(-0.6, 0.6);
+    transform: translateX(170px);
```

Comment:
what is the rational behind the value ? 
ps : the rational cannot be "I saw it on my screen" ;)